From ee25ff3d13f2639b4c3a42125e79f77f921c3320 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Wed, 23 Jan 2019 10:37:37 +0000
Subject: [PATCH] pkg-config: do not add -I to non-existing directory

foo/lib/pgm-5.2/include does not exist, so applications using strict
compiler flags will fail to build due to this -I flag
---
 openpgm/pgm/openpgm-5.2.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/openpgm/pgm/openpgm-5.2.pc.in b/openpgm/pgm/openpgm-5.2.pc.in
index 1e56d944..9e30a6da 100644
--- a/openpgm/pgm/openpgm-5.2.pc.in
+++ b/openpgm/pgm/openpgm-5.2.pc.in
@@ -9,4 +9,4 @@ Version: @PACKAGE_VERSION@
 # packagers may wish to move @LIBS@ to Libs.private for platforms with
 # versions of pkg-config that support static linking.
 Libs: -L${libdir} -lpgm @LIBS@
-Cflags: -I${includedir}/pgm-@VERSION_MAJOR@.@VERSION_MINOR@ -I${libdir}/pgm-@VERSION_MAJOR@.@VERSION_MINOR@/include
+Cflags: -I${includedir}/pgm-@VERSION_MAJOR@.@VERSION_MINOR@
